### ISSUE TITLE
WCAG

### ISSUE THREAD

**Issue by usigna** [not7cd/black-sheep#4]
STRONA GŁÓWNA

1) Brakuje ustawionego języka strony, czyli:
`<html lang="pl">`

2) W miejscach gdzie pojawia się słowo, które wymawia się po angielsku powinno dać się lang="en" dla elementu, czyli np.
```
zamiana z:
<p class="slogan">Restartujemy Hackerspace. Nowa nazwa. Nowy adres. Ci sami ludzie.</p>
na:
<p class="slogan">Restartujemy <span lang="en">Hackerspace</span>. Nowa nazwa. Nowy adres. Ci sami ludzie.</p>
```

3) Brakuje alt dla grafiki loga, dobrze aby opisywał jak wygląda logo wraz z kolorami. Limit znaków dla alt to około 100-110.

4) Tam gdzie są ikonki z "wepnij się" (dla wszystkich):
```
zamiana z:
<i class="las la-door-open"></i>
na:
<span class="las la-door-open" aria-hidden="true"></span>
```

5) Tam gdzie jest statut
```
zamiana z:
<p><a href="/assets/status_hsp.pdf" target="_blank">Statut Stowarzyszenia</a></p>
na:
<p><a href="/assets/status_hsp.pdf" target="_blank">Statut Stowarzyszenia <span class="sr-only">otwórz pdf</span></a></p>
```

Klasa sr-only w css:
`
.sr-only {
          border: 0 !important;
          clip: rect(1px, 1px, 1px, 1px) !important;
          -webkit-clip-path: inset(50%) !important;
          clip-path: inset(50%) !important;
          height: 1px !important;
          margin: -1px !important;
          overflow: hidden !important;
          padding: 0 !important;
          position: absolute !important;
          width: 1px !important;
          white-space: nowrap !important;
        }
`

6) Kontrast w kalendarzu jest zdecydowanie zbyt słaby, czyli ten kolor: #00cc99 jest w nim do zmiany na inny. W tym narzędziu https://whocanuse.com/ jeżeli pokaże się kontrast minimum 4,5:1 to będzie spoko. Mniej jest źle.

7) Focus przy poruszaniu się po stronie za pomocą klawiatury jest zbyt słaby, bo to ten wbudowany w przeglądarkę. Warto dodać tam jakieś obramowanie, zaleca się 2px dotted kolor (np. ten z logo, chociaż często poleca się niebieski).

8) Brakuje "title" w head.

9) Dla "iframe" brakuje "title"
